README.md

```diff
@@ -16,3 +16,14 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+To evaluate a claim like this, there are two main approaches to consider. First, we would want to conduct empirical testing by running numerous tests on the algorithm. This would involve creating various test cases with lists of different lengths and arrangements, such as randomly shuffled lists, already sorted lists, and reverse-sorted lists, among others. During this process, we would measure the runtime to determine if it matches the claimed $O(n)$ complexity. For comparison, we could use the same test cases with other well-known algorithms, such as QuickSort or MergeSort, to see how the proprietary algorithm performs relative to established benchmarks. Additionally, empirical testing should include edge cases, such as empty lists or single-element lists, and stress testing with very large inputs to assess the algorithm's scalability and robustness.
```

Comment:
What would you expect to observe, and how would this tell you whether the claim is correct?

Reply:
What would you expect to observe, and how would this tell you whether the claim is correct?